### ISSUE TITLE
Why is redraw white and the other buttons are blue?

### ISSUE THREAD

**Issue by ljbellis** [chembl/curation-interface#118]
![redraw](https://cloud.githubusercontent.com/assets/9843928/8058437/4afa861a-0eb1-11e5-83c8-888cf95546d6.jpg)

**Comment by mnowotka** [comment]
Because it's a second button and every second button is white. Why? This is a first attempt to assign different colors to different buttons as they perform different actions. For example for sure buttons like 'cancel' or 'delete' should always be last, never selected by default and colored in red. I'm leaving this issue open until the interface will be visually clear. Any ideas of how to do this are welcome.

**Comment by ljbellis** [comment]
Personally, I would rather the first is blue and the rest are white, if we are going to have them coloured.

**Comment by mnowotka** [comment]
Yes, I don't have any preferences, I'm open to all ideas. It doesn't have to be necessary colored by position but it can be colored by function. The problem is choosing the right colors, and not using too many of them.

**Comment by ljbellis** [comment]
Honestly, I am happy with the first being blue, as it tends to be the main function or most important and then the others being white. By only having one white, in my mind, this makes that function less important when in this case, redraw is very useful.

**Comment by mnowotka** [comment]
No problem.
